Resources/public/js/views/ez-confirmboxview.js

```diff
@@ -80,6 +85,42 @@ YUI.add('ez-confirmboxview', function (Y) {
         },
 
         /**
+         * Hides of reveal the buttons
+         *
+         * @method _uiButtonDisplay
+         * @protected
+         */
+        _uiUpdateButton: function () {
+            var container = this.get('container').one('.ez-confirmbox-tools');
+
+            if (this.get('button') === true) {
+                container.show();
```

Comment:
to make that easier to test, if would actually add or remove a class on the container to show/hide the corresponding toolbar